devbox.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://raw.githubusercontent.com/jetify-com/devbox/0.12.0/.schema/devbox.schema.json",
+  "env": {
+    "DEVBOX_COREPACK_ENABLED": "true",
+    "PLAYWRIGHT_SKIP_VALIDATE_HOST_REQUIREMENTS": "true"
+  },
+  "packages": [
+    "nodejs@18.19.0",
+    "nodePackages.pnpm@8.6.12",
```

Comment:
`nodePackages.pnpm` was removed in https://github.com/AmazeeLabs/silverback-template/pull/341/commits/4951fe41906b4a7819c3d772df1553a0aabe8004